erpnext_documentation/www/docs/user/manual/en/human-resources/attendance.md

```diff
@@ -54,6 +54,17 @@ In case the attendance for some employees is not marked, you can mark them as pr
 
     <img class="screenshot" alt="Attendance" src="{{docs_base_url}}/assets/img/human-resources/marking_unmarked_attendance.gif">
 
+> Introduced in Version 13
+
+### 3.2 Remaining Half-day Status
+After Version 12, ERPNext started tracking the remaining half day status. On selecting Half Day [Attendance Status](/docs/user/manual/en/human-resources/attendance-status). Field **Remaining Half day Status** will appear, where you can select the Present or Absent Attendance Status for the remaining half day.
```

Comment:
```suggestion
In ERPNext version 13, you can track the remaining half-day status. On selecting Half Day [Attendance Status](/docs/user/manual/en/human-resources/attendance-status), a field called **Remaining Half Day Status** will be shown, where you can select the Attendance Status as "Present" or "Absent" for the remaining half day.
```

Reply:
Actually, there is a know hardcoded status for attendance status. need to discuss.